addons/mail/wizard/mail_activity_schedule.py

```diff
@@ -150,21 +150,26 @@ def _compute_plan_id(self):
     def _compute_plan_date(self):
         self.plan_date = False
 
+    def _get_summary_lines(self, templates):
```

Comment:
Function should probably be moved at the end of the file with "tools" (to respect sorting/ordering of this file)